.travis.yml

```diff
@@ -49,8 +52,8 @@ matrix:
         - env: DJANGO_VERSION=">=1.9,<1.10"
         - python: "pypy"
 
-services:
-    - elasticsearch
+#services:
```

Comment:
1.7 and 2.x seems great to me - the main thing I was thinking is e.g. AWS with the hosted 1.7 used by many people